aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>Don't think I've seen this approach before. I wonder if there's a more elegant route that does more with showing what data is there and how to extract it. This seems very hard-coded and brittle. If we're not going to play with the data, why not preprocess it, host it on S3, and just have the customer download it in the structure you want it to be in?</p><p> </p><p>Also I'm curious why the use of environment variables. It's not discussed why you're doing that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Environment variables are used for directories/folders. It's done so that a user can possibly also use a terminal to inspect what is present in those folders. (or run linux commands from the notebook itself)</p>